ui/pages/confirmations/components/confirm/footer/footer.tsx

```diff
@@ -225,6 +226,7 @@ const Footer = () => {
     }
     dispatch(updateCustomNonce(''));
     dispatch(setNextNonce(''));
+    dispatch(clearConfirmTransaction());
```

Comment:
Minor, but could we create a local function to remove the duplication such as `resetTransactionState`?